use-cases/meeting-scheduling/src/main/java/org/acme/meetingschedule/rest/DemoDataGenerator.java

```diff
@@ -0,0 +1,255 @@
+package org.acme.meetingschedule.rest;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import ai.timefold.solver.core.impl.util.MutableInt;
+import ai.timefold.solver.core.impl.util.Pair;
```

Comment:
Let's not use internal classes from the solver.
If you need a `Pair`, write the record yourself - it's trivial.
The `MutableInt` is unnecessary.